xonsh/xoreutils/_posix.c

```diff
@@ -0,0 +1,122 @@
+#include <Python.h>
+#include <stdlib.h>
+#include <string.h>
+#include <utmpx.h>
+#ifdef __APPLE__
+#include <sys/sysctl.h>
+#endif
+#include <sys/time.h>
+
+
+/*
```

Comment:
Is this file used anywhere in the code?

Reply:
Normaly only by OsX, but i haven´t OsX for make any tests.

The official library use it for OsX if i remenber well